content/docs/getting_started/reconciler.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Reconciler Guide"
+description: "Reconciler Guide"
+aliases: ["/docs/reconciler_guide"]
+type: docs
+weight: 5
+---
+
+# Reconciler Guide
+
+This guide describes how to enable the reconciler in Open Service Mesh (OSM).
+
+## How the reconciler works
+
+The goal of building a reconciler in OSM is to ensure resources required for the correct operation of OSM's control plane are in their desired state at all times. Resources that are installed as a part of OSM install and have the labels `openservicemesh.io/reconcile: true` and `app.kubernetes.io/name: openservicemesh.io` will be reconciled by the reconciler.
```

Comment:
Do we need to document the `app.kubernetes.io/name` label in this comment, given that it is always added by either the Helm chart or from within the control plane? If it's not exposed to the user, let's not document it.

Reply:
Well this will be added to the mutating webhook and validation webhook and not the CRDs, hence I deliberately left it out

Reply:
Okay, as a user I was just confused why that specific label is mentioned here. Will leave it up to you to decide what's best.